test/JDBC/input/xml/xml_exist-before-16_5-vu-prepare.sql

```diff
@@ -0,0 +1,103 @@
+CREATE TABLE babel_5222_xml_exist_t1 (XmlColumn XML)
+GO
+
+INSERT INTO babel_5222_xml_exist_t1
+VALUES ('<artists> <artist name="John Doe"/> <artist name="Edward Poe"/> <artist name="Mark The Great"/> </artists>')
+GO
+
+CREATE TABLE babel_5222_xml_exist_t2 (Id INT PRIMARY KEY, XmlColumn XML);
+GO
+
+INSERT INTO babel_5222_xml_exist_t2 (Id, XmlColumn)
+VALUES (1, '<Root><Child1>Value1</Child1></Root>'),
+       (2, '<Root><Child2>Value2</Child2></Root>'),
+       (3, '<Root><Child1>Value1</Child1><Child2>Value2</Child2></Root>');
+GO
+
+CREATE TABLE babel_5222_xml_exist_text (XmlColumn TEXT)
+GO
+
+INSERT INTO babel_5222_xml_exist_text
+VALUES ('<artists> <artist name="John Doe"/> <artist name="Edward Poe"/> <artist name="Mark The Great"/> </artists>')
+GO
+
+CREATE TYPE dbo.babel_5222_xml_exist_varcharUDT FROM VARCHAR(100);
+GO
+
+CREATE TYPE dbo.babel_5222_xml_exist_imageUDT FROM IMAGE;
+GO
```

Comment:
how about UDF created from `xml`?

Reply:
Creation of UDT on `xml` is not allowed in TSQL, hence not added test for the same.